hazelcast/src/main/java/com/hazelcast/spi/Operation.java

```diff
@@ -182,9 +182,13 @@ public final long getCallId() {
     // Accessed using OperationAccessor
     final Operation setCallId(long callId) {
         this.callId = callId;
+        onSetCallId();
         return this;
     }
 
+    protected void onSetCallId() {
```

Comment:
Can you add a line of comment what the purpose is of this method and when it is expected to be called?